docs/src/main/sphinx/security/security-overview.rst

```diff
@@ -0,0 +1,159 @@
+=================
+Security overview
+=================
+
+After the initial :doc:`installation </installation>` of your cluster, security
+is the next major concern for successfully operating Trino. This overview
+provides an introduction to different aspects of configuring security for your
+Trino cluster.
+
+Aspects of configuring security
+-------------------------------
+
+The default installation of Trino has no security features enabled. Security
+can be enabled for different parts of the Trino architecture:
+
+* :ref:`security-client`
+* :ref:`security-inside-cluster`
+* :ref:`security-data-sources`
+
+Suggested configuration workflow
+--------------------------------
+
+To configure security for a new Trino cluster, follow this best practice
+order of steps. Do not skip or combine steps.
+
+1. **Enable** HTTPS
```

Comment:
Bold for the whole title?

Reply:
@electrum and @dain .. we really need to get the TLS docs merge so this can be a link to all the new great content.

Reply:
Bolded.